bin/start_aws.sh

```diff
@@ -181,7 +181,7 @@ if pulumi config get sirius:ledger_pwd -C ${script_dir}/../pulumi/python/kuberne
   true
 else
   LEDGER_PW=$(createpw)
-  pulumi config set --secret sirius:accounts_pwd -C ${script_dir}/../pulumi/python/kubernetes/applications/sirius $LEDGER_PW
+  pulumi config set --secret sirius:ledger_pwd -C ${script_dir}/../pulumi/python/kubernetes/applications/sirius $LEDGER_PW
```

Comment:
oops, looks like I missed this in my previous review.  Sorry about that!

Reply:
No worries; this actually ties into your Jenkinsfile question below which is why my automated tools missed it....